docs/operator-manual/user-management/microsoft.md

```diff
@@ -92,107 +80,125 @@
-                     clientID: {azure_ad_application_client_id}
-                     clientSecret: $oidc.azure.clientSecret
-                     requestedIDTokenClaims:
-                        groups:
-                           essential: true
-                     requestedScopes:
-                        - openid
-                        - profile
-                        - email
+        data:
+          url: https://argocd.example.com/ # Replace with the external base URL of your Argo CD
+          oidc.config: |
+            name: Azure
+            issuer: https://login.microsoftonline.com/{directory_tenant_id}/v2.0
+            clientID: {azure_ad_application_client_id}
+            clientSecret: $oidc.azure.clientSecret
+            requestedIDTokenClaims:
+               groups:
+               essential: true
+            requestedScopes:
+               - openid
+               - profile
+               - email
 
-2. Edit `argocd-secret` and configure the `data.oidc.azure.clientSecret` section:
+2. Edit the `argocd-secret` Secret and configure the `data.oidc.azure.clientSecret` section:
 
-            Secret -> argocd-secret
+         data:
+           oidc.azure.clientSecret: {client_secret | base64_encoded}
 
-            data:
-               oidc.azure.clientSecret: {client_secret | base64_encoded}
+### Azure AD App Registration Auth using Dex
 
-3. Edit `argocd-rbac-cm` to configure permissions. Use group ID from Azure for assigning roles
-      [RBAC Configurations](../rbac.md)
+Configure a new AD App Registration, as above.
+Then, add the `dex.config` to the `argocd-cm` ConfigMap:
 
-            ConfigMap -> argocd-rbac-cm
+```yaml
+data:
+  dex.config: |
+    connectors:
+    - type: microsoft
+      id: microsoft
+      name: Your Company GmbH
+      config:
+        clientID: $MICROSOFT_APPLICATION_ID
+        clientSecret: $MICROSOFT_CLIENT_SECRET
+        redirectURI: http://localhost:8080/api/dex/callback
+        tenant: ffffffff-ffff-ffff-ffff-ffffffffffff
+        groups:
+          - argocd_access_SG
+          - argocd_org_admin_SG
+          - argocd_admin_SG
+```
 
-            policy.default: role:readonly
-            policy.csv: |
-               p, role:org-admin, applications, *, */*, allow
-               p, role:org-admin, clusters, get, *, allow
-               p, role:org-admin, repositories, get, *, allow
-               p, role:org-admin, repositories, create, *, allow
-               p, role:org-admin, repositories, update, *, allow
-               p, role:org-admin, repositories, delete, *, allow
-               g, "84ce98d1-e359-4f3b-85af-985b458de3c6", role:org-admin
+## Configuring authorization through RBAC
+Edit the `argocd-rbac-cm` ConfigMap to assign permissions from the user's Azure AD security group memberships. After being logged in into ArgoCD, check [the user's group membership](log-in-to-argocd-ui-using-sso) to decide if you have to use the security group's `Object ID` or `name`.
```

Comment:
s/ArgoCD/Argo CD

Reply:
Fixed (and other occurences without whitespace)